### ISSUE TITLE
unittests is outdated?

### ISSUE THREAD

**Issue by denizzzka** [dymk/temple#28]
Used test string from travis-ci script:

```
$ dub --build=unittest --config=unittest 
Target vibe-d 0.7.24 is up to date. Use --force to rebuild.
Building temple 0.7.3+commit.4.g66def3e configuration "unittest", build type unittest.
Compiling using dmd...
Compiling Temple with Vibed support
src/temple/temple.d(211): Error: static assert  (isOutputRange!(OutputStream, string)) is false
FAIL .dub/build/unittest-unittest-linux.posix-x86_64-dmd_2067-B5243AF6FD7D2C79B1473483C8C3DDA6/ temple executable
Error executing command run:
dmd failed with exit code 1.
```

**Comment by denizzzka** [comment]
dmd 2.067.0

**Comment by denizzzka** [comment]
double

**Comment by dymk** [comment]
Fixed with #30
